jbpm-persistence/jbpm-persistence-api/src/main/java/org/jbpm/persistence/api/integration/model/ProcessInstanceView.java

```diff
@@ -186,7 +186,7 @@ public void copyFromSource() {
         this.correlationKey = ((WorkflowProcessInstanceImpl)source).getCorrelationKey();
         this.date = new Date();
         this.id = source.getId();
-        this.initiator = (String) ((WorkflowProcessInstance)source).getMetaData().get("Initiator");
+        this.initiator = (String) ((WorkflowProcessInstanceImpl)source).getVariable("initiator");
```

Comment:
What is the difference between these two approaches?

Reply:
variable is always available while meta data only on start of process instance - since meta data is not persisted.